src/content/config.ts

```diff
@@ -55,10 +59,8 @@ export const collections = {
 					'A date string or YAML date that is compatible with JavaScript’s `new Date()` constructor.',
 				),
 			authors: z
-				.array(
-					// biome-ignore format:
-					z.enum(['astro-team', 'ben', 'bjorn', 'chris', 'dan', 'drew', 'elian', 'ema', 'erika', 'fred', 'fuzzy', 'hideoo', 'jon', 'mark', 'martrapp', 'matt', 'matthew', 'nate', 'reuben', 'sarah', 'thuy', 'tony', 'yan']),
-				)
+				.enum(Object.keys(authors) as [keyof typeof authors, ...(keyof typeof authors)[]])
+				.array()
```

Comment:
Switching authors to a single JSON file makes it easy to use an `enum` here which is locked to the file keys rather than maintaining this list twice.